Modix/Modules/UserInfoModule.cs

```diff
@@ -0,0 +1,122 @@
+        public async Task GetUserInfo([Remainder] IUser user = null)
+        {
+            user = user ?? Context.User;
+
+            var utcNow = DateTime.UtcNow;
+            var builder = new StringBuilder();
+            builder.AppendLine("**\u276F User Information**");
+            builder.AppendLine("ID: " + user.Id);
+            builder.AppendLine("Profile: " + user.Mention);
+
+            // TODO: Add content about the user's presence, if any
+
+            builder.AppendFormat(
+                CultureInfo.InvariantCulture,
+                "Created: {0} ago ({1:yyyy-MM-ddTHH:mm:ssK})\n",
+                (utcNow - user.CreatedAt).Humanize(maxUnit: TimeUnit.Year, culture: CultureInfo.InvariantCulture),
+                user.CreatedAt.UtcDateTime);
+
+            if (user is IGuildUser member)
+            {
+                builder.AppendLine();
+                builder.AppendLine("**\u276F Member Information**");
+
+                if (!string.IsNullOrEmpty(member.Nickname))
+                {
+                    builder.AppendLine("Nickname: " + member.Nickname);
+                }
+
+                if (member.JoinedAt is DateTimeOffset joinedAt)
+                {
+                    builder.AppendFormat(
+                        CultureInfo.InvariantCulture,
+                        "Joined: {0} ago ({1:yyyy-MM-ddTHH:mm:ssK})\n",
+                        (utcNow - joinedAt).Humanize(maxUnit: TimeUnit.Year, culture: CultureInfo.InvariantCulture),
+                        joinedAt.UtcDateTime);
+                }
+
+                if (member.RoleIds.Count > 0)
+                {
+                    var roles = member.RoleIds.Select(x => member.Guild.Roles.Single(y => y.Id == x))
+                        .Where(x => !string.Equals("@everyone", x.Name, StringComparison.Ordinal))
+                        .OrderBy(x => x.Name, StringComparer.InvariantCultureIgnoreCase)
+                        .ToArray();
+
+                    if (roles.Length > 0)
+                    {
+                        Array.Sort(roles, StringComparer.OrdinalIgnoreCase);
+                        builder.AppendLine("Roles: " + string.Join(',', (object[])roles));
+                    }
+                }
+            }
+
+            // TODO: Add infraction summary
+
+            // TODO: Add voice session data
+
+            var embedBuilder = new EmbedBuilder
+            {
+                Author = new EmbedAuthorBuilder
+                {
+                    Name = user.Username + "#" + user.Discriminator,
+                    IconUrl = user.GetAvatarUrl()
+                },
+                Color = GetDominateColor(user),
+                Description = builder.ToString(),
+                ThumbnailUrl = user.GetAvatarUrl()
+            };
+
+            await ReplyAsync(string.Empty, embed: embedBuilder.Build());
+        }
+
+        [Command("info")]
+        public async Task GetUserInfo([Remainder] ulong userId)
+        {
```

Comment:
[Remainder] is only needed if you expect the parameter to accept a string (with spaces).

Reply:
Fixed in f509448.